Makefile

```diff
@@ -6,19 +6,18 @@
 #     .\_/.
 #
 
-prepare:
-	kubectl create ns aegis-system
-
 clean:
-	kubectl delete ns aegis-system
-	kubectl delete deployment aegis-workload-demo -n default
-
-install:
-	echo "Not implemented yet!"
-
-clean-prepare-all: clean prepare-all
+	@if kubectl get ns | grep aegis-system; then \
+		kubectl delete ns aegis-system; \
+		kubectl delete deployment aegis-workload-demo -n default; \
+	else \
+  		echo "Nothing to clean."; \
+	fi
```

Comment:
This if/else take care of the error `kubectl` spits out if the namespace does not exist.

If the `aegis-system` namespace is not there, we assume that someone/something has already cleaned things up.